### PR TITLE
adding try catch to processMerge function

### PR DESCRIPTION
**Pull request by codebird**
closes https://github.com/flutter/flutter/issues/102317

Added try catch around the github merge graphql call, so the process goes on and finishes other PRs

### CODE REVIEW

File: auto_submit/lib/requests/check_pull_request.dart [hunk 1]
```diff
@@ -127,17 +127,21 @@ class CheckPullRequest extends AuthenticatedRequestHandler {
...
+          'oid': sha,
+          'title': '$title (#$number)',
+        },
+      ));
+      if (result.hasException) {
+        log.severe('Failed to merge pr#: $number with ${result.exception.toString()}');
+        return false;
+      }
+    } catch (e) {
+      log.severe('_processMerge error in $slug: $e');
```
Comment (CaseyHillers) [inline auto_submit/lib/requests/check_pull_request.dart:144]: Can we add test for this?

Comment (codebird) [reply auto_submit/lib/requests/check_pull_request.dart:144]: @CaseyHillers can't figure out a way to trigger the exception :) I'll keep trying, but any clue would be helpful.

Comment (codebird) [reply auto_submit/lib/requests/check_pull_request.dart:144]: looking

Comment (codebird) [reply auto_submit/lib/requests/check_pull_request.dart:144]: Thanks for the help @CaseyHillers

**Review by CaseyHillers — APPROVED** [review]

**Review by keyonghan — APPROVED** [review]
LGTM

**Pull request merged**